### PR TITLE
User profile projects

### PR DESCRIPTION
**Pull request by veiga91**

### CODE REVIEW

**Comment by veiga91** [comment]
I rebased but the commit d8af527 persists.

File: packages/app/client/src/reduxStructures/profile/dataManipulation.js [hunk 1]
```diff
@@ -8,3 +8,40 @@ export const onSaveInfos = (state, payload) => {
     badges: userBadges
   }
 }
+// TODO: MAKE THE FUNCTIONS BELOW AN UTILITY BECAUSE THEIR LOGIC HAVE BEEN USED ON PROJECT REDUCER AS WELL
```
Comment (campbell-codes) [inline packages/app/client/src/reduxStructures/profile/dataManipulation.js:11]: Not keen on todos in code we should use zube to track things that need to be done. You can link to the line in the code there

**Review by campbell-codes — APPROVED** [review]
One comment

File: packages/app/client/src/reduxStructures/profile/dataManipulation.js (hunk 1 above)
Comment (veiga91) [reply packages/app/client/src/reduxStructures/profile/dataManipulation.js:11]: will do!

**Pull request merged**